test/extensions/filters/common/rbac/BUILD

```diff
@@ -0,0 +1,40 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_mock",
+    "envoy_cc_test",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_test(
```

Comment:
All of these tests should be defined using envoy_extension_cc_test(). I think I see why you didn't use it here, since there is no clear extension that owns it if it gets used from multiple filters. I would for now have it be owned by the HTTP rbac filter and put in a TODO That we might need to make a test owner a list of multiple owners so that if either the network or HTTP filter gets included we test this stuff.

Reply:
I was being consistent with the other libs in the common namespace. I'll make this change and add the TODO.

Reply:
Also adding a `envoy_extension_cc_mock` with similar mechanics.